common/src/main/resources/locales/tr-tr.yml

```diff
@@ -0,0 +1,173 @@
+edit_home_menu_privacy_button_public: '[[☀ Herkese Açık Yap…]](#6fff00 show_text=&7Bu evi herkese açık yapmak için tıklayın run_command=/huskhomes:edithome %1% privacy -m)'
+edit_home_menu_privacy_button_private: '[[∅ Özel Yap…]](#ff9f0f show_text=&7Bu evi özel yapmak için tıklayın run_command=/huskhomes:edithome %1% privacy -m)'
+edit_home_menu_meta_edit_buttons: '[Düzenle:](gray) [[✎ Yeniden isimlendir…]](#c160ff-#bb50ff show_text=&7Yeniden adlandırma komutunu önermek için tıklayın suggest_command=/huskhomes:edithome %1% rename )   [[ℹ Açıklama…]](#bb50ff-#b43fff show_text=&7Açıklamayı düzenle komutunu önermek için tıklayın suggest_command=/huskhomes:edithome %1% description )'
+edit_home_privacy_public_success: '[%1%](#00fb9a bold) [eviniz başarıyla halka açıldı](#00fb9a)'
+edit_home_privacy_public_success_other: '[%1% adlı oyuncunun evi](#00fb9a) [%2%](#00fb9a bold) [başarıyla herkese açıldı.](#00fb9a)'
+edit_home_privacy_private_success: '[%1%](#00fb9a bold) [eviniz başarıyla özelleştirildi.](#00fb9a)'
+edit_home_privacy_private_success_other: '[%1% adlı oyuncnun evi başarıyla](#00fb9a) [%2%](#00fb9a bold) [özel yapıldı.](#00fb9a)'
+edit_home_update_location: '[%1%](#00fb9a bold) [eviniz mevcut konumunuza yeniden yerleştirildi.](#00fb9a)'
+edit_home_update_location_other: '[%1% adlı oyuncunun evi](#00fb9a) [%2%](#00fb9a bold)[mevcut konumunuza yeniden yerleştirildi.](#00fb9a)'
+edit_home_update_description: '[Evinizin açıklaması başarıyla değiştirildi, %1%:](#00fb9a) \n[←](white) [Eski:](#00fb9a) ["%2%"](gray)\n[→](white) [Yeni:](#00fb9a) ["%3%" ](gray)'
+edit_home_update_description_other: '[%1% adlı oyuncunun evinin açıklaması başarıyla değiştirildi, %2%:](#00fb9a) \n[←](white) [Eski:](#00fb9a) ["%3%"](gray)\n[→](white) [Yeni:](#00fb9a) ["%4%" ](gray)'
+edit_home_update_name: '[Eviniz başarıyla yeniden adlandırıldı: %1% →](#00fb9a) [%2%](#00fb9a bold)'
+edit_home_update_name_other: '[%1% adlı oyuncunun evinin adı başarıyla değiştirildi: %2% →](#00fb9a) [%3%](#00fb9a bold)'
+edit_warp_menu_title: '[%1%](#00fb9a bold) [warpının ayrıntıları gösteriliyor:](#00fb9a)'
+edit_home_menu_metadata: '[⌚ %1%](#999999-#ababab show_text=&7Zaman Damgasını Ayarla:\n&8Warp ayarlandıysa)   [⚡ %2%](#ababab-#c7c7c7 show_text=&7Warp ID:\n&8%3%)'
+edit_warp_menu_description: '[ℹ %1%](#c160ff-#b43fff show_text=&#c160ff&Açıklama:\n&8%2%)\n'
+edit_warp_menu_world: '[⭘ %1%](#3b76ff-#5286ff show_text=&#3b76ff&Dünya:\n&8Warpın ayarlandığı dünya'
+edit_warp_menu_world_server: '[⭘ %1%](#3b76ff-#477fff show_text=&#3b76ff&Dünya:\n&8Warpın ayarlandığı dünya)   [☁ %2%](#477fff-#5286ff show_text=&#3b76ff&Sunucu:\n&8Warpın ayarlandığı sunucu)'
+edit_warp_menu_coordinates: '[⚐ x: %1%, y: %2%, z: %3%](#ffc43b-#f5c962 show_text=&#ffc43b&Koordinatlar:\n&8Warpın dünyadaki konumu\n&8yaw: %4%, pitch: %5%)\n'
+edit_warp_menu_use_buttons: '[Use:](gray) [[⏩ Işınlanma…]](#00fb9a show_text=&7Bu warp'a ışınlanmak için tıklayın run_command=/huskhomes:warp %1%)'
```

Comment:
This is failing because there's a floating single quote mark (`'`) in the middle of the text. You need to escape this with a double `''`
```suggestion
edit_warp_menu_use_buttons: '[Use:](gray) [[⏩ Işınlanma…]](#00fb9a show_text=&7Bu warp''a ışınlanmak için tıklayın run_command=/huskhomes:warp %1%)'
```